tranlation_script/translate_queries.py

```diff
@@ -0,0 +1,94 @@
+"""
+Script is based on this package: https://github.com/nidhaloff/deep-translator/tree/a694c92b6741fc9c3200835b64be2fd910cd761b
+How to use: python tranlation_script/translate_queries.py --questions_file_path=tranlation_script/test.csv --source=hausa --pivot=english --output_file_path=tranlation_script/twi_translated.csv
+"""
+
+import os
+import argparse
+import pandas as pd
+from tqdm.auto import tqdm
+from deep_translator import GoogleTranslator
+
+TARGET_LANG = {
+    "bem": "auto",
+    "fon": "auto",
+    "hausa": "ha",
+    "igbo": "ig",
+    "kinyarwanda": "rw",
+    "twi": "ak",
+    "yoruba": "yo",
+    "swahili": "sw",
+    "wolof": "auto",
+    "zulu": "zu",
+    "english": "en",
+}
+
+
+def get_parser() -> argparse.ArgumentParser:
+    parser = argparse.ArgumentParser(
+        description="Translate source language to pivot language"
+    )
+    parser.add_argument(
+        "--questions_file_path",
+        type=str,
+        required=True,
+        help="File path to the file containing the source language data",
+    )
+    parser.add_argument(
+        "--source",
+        type=str,
+        required=True,
+        help="original language of the data to be translated",
+        choices=TARGET_LANG,
+    )
+    parser.add_argument(
+        "--pivot",
+        type=str,
+        required=True,
+        default="en",
```

Comment:
can we restrict options here to "fr" and "en"

Reply:
Yes. I'll do that now.